Source/santad/EventProviders/SNTEndpointSecurityProcessFileAccessAuthorizer.mm

```diff
@@ -54,6 +57,7 @@ - (instancetype)initWithESAPI:(std::shared_ptr<santa::EndpointSecurityAPI>)esApi
     _iterateProcessPoliciesBlock = iterateProcessPoliciesBlock;
 
     _procRuleCache = std::make_unique<ProcessRuleCache>(2000);
+    _configurator = [SNTConfigurator configurator];
```

Comment:
This doesn't seem to be used anywhere?

Reply:
It's used to get the configured override action below.

Reply:
Oh I missed that. What's the benefit of having it as a property?

Reply:
Previous times I've generated and looked at flame graphs, there was a surprisingly large amount of time spent retrieving the shared config object over and over. So I generally grab it once for the object and save the call.